lib/general.sh

```diff
@@ -116,7 +116,7 @@ exit_with_error()
 
 get_package_list_hash()
 {
-	( printf '%s\n' $PACKAGE_LIST | sort -u; printf '%s\n' $PACKAGE_LIST_EXCLUDE | sort -u; echo "$ROOTFSCACHE_VERSION" ) \
+	( printf '%s\n' $PACKAGE_LIST | sort -u; printf '%s\n' $PACKAGE_LIST_EXCLUDE | sort -u; echo "$ROOTFSCACHE_VERSION"; echo "${REVISION:0:3}") \
```

Comment:
This is not needed. Move out.

Reply:
You mean to make it reluctant to Armbian revision while sensitive to selected package set?

Reply:
$ROOTFSCACHE_VERSION already do the job REVISION dependency is not needed - to force unique hash on the same package set. Its rarely needed. I added REVISION by mistake / overlooked.